nuget.config

```diff
@@ -4,6 +4,7 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
-	<add key="toolkit-labs" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json" />
+    <add key="toolkit-labs" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json" />
+    <add key="computesharp-preview" value="https://pkgs.computesharp.dev/index.json" />
```

Comment:
Please publish to regular nuget instead

Reply:
You'll have to wait then 😛
And a stable release isn't coming for at least another couple months (it's tied to Win2D's release).